docs/stories/getting started/changelog.mdx

```diff
@@ -0,0 +1,23 @@
+import { Meta } from '@storybook/blocks';
+
+import PrimitivesChangelog from '../../../packages/primitives/CHANGELOG.md?raw';
+import DSChangelog from '../../../packages/strapi-design-system/CHANGELOG.md?raw';
+import IconsChangelog from '../../../packages/strapi-icons/CHANGELOG.md?raw';
+
+<Meta title="Getting Started/Changelog" />
+
+# Changelog
+
+This is autogenerated by [Changesets](https://github.com/changesets/changesets)
+
+<div style={{ paddingBottom: 20 }} />
+
+<Markdown>{DSChangelog}</Markdown>
```

Comment:
is it possible to see these as a separate links , otherwise people will have to scroll to bottom to check icons or primitives changelog.

Reply:
didn't realise the Markdown copy didn't have links, so i've just made them separate 👍🏼